<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Login sucessfully executed]]>
    </line>
    <line>
      <![CDATA[<a target="blank" href="H:\selenium practise\error.jpg">Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>Capturing Screenshot<br>]]>
    </line>
    <line>
      <![CDATA[<br>Login sucessfully executed<br>]]>
    </line>
    <line>
      <![CDATA[<a target="blank" href="H:\selenium practise\error.jpg">Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>Capturing Screenshot<br>]]>
    </line>
    <line>
      <![CDATA[<br>Login sucessfully executed<br>]]>
    </line>
    <line>
      <![CDATA[<a target="blank" href="H:\selenium practise\error.jpg">Screenshot</a>]]>
    </line>
  </reporter-output>
  <suite name="Data Driven Basics" duration-ms="11473" started-at="2020-09-18T19:17:18Z" finished-at="2020-09-18T19:17:29Z">
    <groups>
    </groups>
    <test name="BankManagerLoginTest" duration-ms="2129" started-at="2020-09-18T19:17:18Z" finished-at="2020-09-18T19:17:20Z">
      <class name="com.w2a.testcases.BankManagerLoginTest">
        <test-method status="FAIL" signature="loginAsBankManager()[pri:0, instance:com.w2a.testcases.BankManagerLoginTest@477b4cdf]" name="loginAsBankManager" duration-ms="2120" started-at="2020-09-19T00:47:18Z" finished-at="2020-09-19T00:47:20Z">
          <exception class="java.lang.AbstractMethodError">
            <full-stacktrace>
              <![CDATA[java.lang.AbstractMethodError
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1659)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1639)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:703)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AbstractMethodError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsBankManager -->
      </class> <!-- com.w2a.testcases.BankManagerLoginTest -->
    </test> <!-- BankManagerLoginTest -->
    <test name="Add Customer Test" duration-ms="9326" started-at="2020-09-18T19:17:20Z" finished-at="2020-09-18T19:17:29Z">
      <class name="com.w2a.testcases.AddCustomerTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.w2a.testcases.AddCustomerTest@4e08711f]" name="setup" is-config="true" duration-ms="4954" started-at="2020-09-19T00:47:13Z" finished-at="2020-09-19T00:47:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="addCustomer(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.w2a.testcases.AddCustomerTest@4e08711f]" name="addCustomer" duration-ms="9324" started-at="2020-09-19T00:47:20Z" data-provider="getData" finished-at="2020-09-19T00:47:29Z">
          <exception class="java.lang.AbstractMethodError">
            <full-stacktrace>
              <![CDATA[java.lang.AbstractMethodError
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1659)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1639)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:703)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AbstractMethodError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomer -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.w2a.testcases.AddCustomerTest@4e08711f]" name="teardown" is-config="true" duration-ms="604" started-at="2020-09-19T00:47:29Z" finished-at="2020-09-19T00:47:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.w2a.testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
  </suite> <!-- Data Driven Basics -->
</testng-results>
